src/mavsdk/plugins/camera/camera_impl.cpp

```diff
@@ -219,6 +219,31 @@ void CameraImpl::manual_disable()
     _camera_found = false;
 }
 
+void CameraImpl::update_component()
```

Comment:
Where is this called from?

Reply:
It's called on the `CameraImpl::select_camera(const size_t id)` (line 260 of this file)  this method in on the public interface and can be called anytime after the constructor.